src/main/scala/org/constellation/util/APIClient.scala

```diff
@@ -51,6 +51,8 @@ class APIClient private (
   dao: DAO = null
 ) extends APIClientBase(host, port, authEnabled, authId, authPassword) {
 
+  implicit val contextShift: ContextShift[IO] = IO.contextShift(executionContext)
```

Comment:
I thought we want to explicitly use  executors

Reply:
Yes, it was done before EC changes